index.js

```diff
@@ -57,6 +56,16 @@ app.get('/qrcards', (req, res) => {
     .catch((err) => res.status(500).json({error: 'Error in creating QR card.'}))
 })
 
+app.delete('/deleteqr/:id', (req, res) => {
+  const query = knex('qrcodes')
+    .where('id', '=', req.params.id)
+    .del()
+
+  query
+    .then((data) => res.json(data))
```

Comment:
then you should probably just reply with a 200 status code: `res.sendStatus(200)`